jetty-ee10/jetty-ee10-quickstart/src/main/java/org/eclipse/jetty/ee10/quickstart/PreconfigureQuickStartWar.java

```diff
@@ -95,7 +95,7 @@ public static void preconfigure(Resource war, Resource dir, Resource xml) throws
             if (!dir.exists())
                 Files.createDirectories(dir.getPath());
 
-            URI jarUri = URI.create("jar:" + war.getURI() + "!/");
+            URI jarUri = Resource.toJarFileUri(war.getURI());
```

Comment:
`toJarFileUri` seems like it should not be part of `Resource` anymore. Or maybe should go away entirely.

Reply:
Ditto, Lets try to address this in the FileID/URIUtil second PR (not the first)